ifrit98/storage-subnet
# update api, cmds, broadband, python min 3.9

* Updates API to use latest dummy encryption wallet (security)
* Adds badges to README
* Update python min version to 3.9 (`asyncio.to_thread()` introduced in 3.9)
* Make sure tier factor multiplier applies to failed retrieve rewards also
